sim/core/major_cooldown.go

```diff
@@ -486,17 +483,12 @@ restart:
 			goto restart
 		}
 	}
-	mcdm.tryUsing = false
-
 	mcdm.minReady = mcdm.majorCooldowns[0].ReadyAt()
 }
 
 // This function should be called if the CD for a major cooldown changes outside
 // of the TryActivate() call.
 func (mcdm *majorCooldownManager) UpdateMajorCooldowns() {
-	if mcdm.tryUsing {
```

Comment:
afaik it shouldn't be a big deal if this happens, and occurs for tricks because tricks needs to update MCD on its cast